packages/flutter/test/painting/image_provider_test.dart

```diff
@@ -158,6 +158,7 @@ void main() {
 
         Future<void> loadNetworkImage() async {
           final NetworkImage networkImage = NetworkImage(nonconst('foo'));
+          // ignore: invalid_use_of_protected_member
```

Comment:
@hterkelsen This is the error I'm ignoring with the more specific import, didn't figure it out right away